tomcat-9.0/src/main/java/org/dogtagpki/jss/tomcat/JSSContext.java

```diff
@@ -41,6 +40,19 @@ public JSSContext(String alias) {
         }
     }
 
+    public JSSContext(String alias) {
+        this();
+        logger.debug("JSSContext(" + alias + ")");
+        aliases = new ArrayList<>();
+        aliases.add(alias);
+    }
+
+    public JSSContext(List<String> aliases) {
+        this();
```

Comment:
I think String.join() below will throw NPE if aliases is null.  Considering checking 
if (aliases == null || aliases.isEmpty()) {